src/types/Todo.ts

```diff
@@ -0,0 +1,9 @@
+import { User } from './User';
```

Comment:
Consider sorting imports alphabetically for better readability and maintainability. 📚